roxctl/common/client.go

```diff
@@ -21,6 +23,12 @@ import (
 
 var (
 	http1NextProtos = []string{"http/1.1", "http/1.0"}
+
+	// RoxctlCommand is the reconstructed roxctl command line.
+	RoxctlCommand string
```

Comment:
I'm wondering if we really need this global variable or if we could hide it in the client somehow. But unfortunately I don't see an easy way to do it without complicating the pre command hook.

Reply:
I share your dislike of global variables.